allow downgrade of packages on host image generation
(required for flexible switching version in nfs root)

diff --git a/package/Makefile b/package/Makefile
--- a/package/Makefile
+++ b/package/Makefile
@@ -80,6 +80,7 @@ OPKG:= \
 	--force-overwrite \
 	--force-postinstall \
 	--force-maintainer \
+	--force-downgrade \
 	--add-dest root:/ \
 	--add-arch all:100 \
 	--add-arch $(if $(ARCH_PACKAGES),$(ARCH_PACKAGES),$(BOARD)):200
